backend/members/lookup.go

```diff
@@ -3,9 +3,24 @@ package members
 import (
 	"context"
 	"fmt"
+	"github.com/cloudevents/sdk-go/v2/event/datacodec/json"
```

Comment:
Any specific reason for not using the builtin json package?

Reply:
no. good spot. autoimport 🤷